### DISCUSSION TITLE
Length and Width of arrow indicating direction

### DISCUSSION THREAD

**Discussion by galaxyeee** [theislab/scvelo#915] [Q&A] [answered]
Dear scvelo developers,

It may be a basic question, but I am writing an issue because I have a question.

<img width="470" alt="image" src="https://user-images.githubusercontent.com/101790387/184047379-7b05879b-085f-4c57-8ec1-186c2fc1406c.png">

In Figure, I knew the longer the arrow indicating direction, the faster the differentiation to the next cell cluster. But can you tell me more detail about the meaning of the length and the width of the arrow?
I want to know how the longer the length, the thicker the width, different the meaning.

Thank you.

Best,
EUNHA LIM

**Comment by galaxyeee** [comment]
Hi @WeilerP,

I haven't found the answer to the question yet, so I would really appreciate it if you could answer it.
Thank you.

Best,
EUNHA LIM

**Comment by WeilerP** [comment, marked as answer]
@galaxyeee, the width should correspond to the magnitude of the velocity.
